operator/internal/deploy/chart_manager.go

```diff
@@ -18,24 +18,26 @@ import (
 	k8syaml "sigs.k8s.io/yaml"
 )
 
-type SKRChartManager struct {
+type SKRWebhookChartManager struct {
 	webhookChartPath             string
 	skrWebhookMemoryLimits       string
 	skrWebhookCPULimits          string
 	kcpAddr                      string
 	enableWebhookPreInstallCheck bool
 }
 
-func NewSKRChartManager(chartPath, memoryLimits, cpuLimits string, enableWebhookPreInstallCheck bool) *SKRChartManager {
-	return &SKRChartManager{
+func NewSKRWebhookChartManager(chartPath, memoryLimits, cpuLimits string,
```

Comment:
I think memoryLimits and cpuLimits are probably not necessary to be put as separate args. Maybe we create a config Struct for this manager?